client/me/security-account-email/index.tsx

```diff
@@ -48,9 +48,18 @@ const SecurityAccountEmail = ( { path }: { path: string } ) => {
 			.then( () => {
 				markSaved();
 
+				const email = ( unsavedUserSettings?.user_email as string ) || '';
+
 				dispatch(
 					successNotice(
-						translate( 'Your account email address was updated successfully.' ),
+						translate(
```

Comment:
This is for the notice shown when saving the email change on /me/security/account-email